verilog/analysis/checkers/struct_union_name_style_rule.h

```diff
@@ -51,6 +54,8 @@ class StructUnionNameStyleRule : public verible::SyntaxTreeLintRule {
   static const char kMessageStruct[];
   static const char kMessageUnion[];
 
+  std::string raw_tokens;
```

Comment:
Make sure that instance variables [have a trailing underscore](https://google.github.io/styleguide/cppguide.html#Variable_Names)